docu/Concepts/Snippets/1-Blockchain/Roles.md

```diff
@@ -0,0 +1,53 @@
+# Roles
+User Roles also handled by blockchain and node servers
+
+My Goal is to save not only the gradido transactions in blockchain but also 
+who is allowed to create new gradidos,
+who allow joining new user to the group,
+who allow connect to another group and 
+how the community decide which one is allowed to do this things.
+
+## Why?
+If this would be handled only by community-server everyone could be easly 
+overwrite this rules by using a modified client to send his transactions direct over
+hedera or iota bypassing the community-server rules. 
+With hedera it is possible to only allow sending messages to a topic with the correct admin key,
+but then is the admin the single point of failure. Also must the key saved on server to allow everyone 
+sending gradidos or the transactions will only be proccessed, when admin is logging in.
+If we don't use blockchain technologie at all, we have a big single point of failure. 
+The Community-Server and everyone who has direct access to server and the admins of course.
+But it would be much much simpler of course :)
```

Comment:
Aussage: Wenn wir keine Blockchain benutzen, haben wir einen großen Single-Point-Of-Failure.

Das ist nicht korrekt, wir haben viele kleine Points-Of-Failure, da wir ja mehrere Communities federieren möchten.
![image](https://user-images.githubusercontent.com/1238238/131407846-9b0034c0-5950-4718-a176-f300dca0494e.png)
Wir können uns nun vorstellen, dass Knoten B korrumpiert ist - d.h. z.B. ist der Server abgestürzt oder der Admin schließt Leute aus bzw ist korrupt. Der Rest des Netzwerkes bleibt allerdings intakt, er beeinflusst die anderen Knoten nur durch Schöpfung (z.B. kann er versuchen das Netzwerk mit "illegal" geschöpften Gradidos zu fluten)

Reply:
Wenn wir individuelle "Community-Gradidos" haben, können illegale Gradidos über Blacklist gesperrt werden.